### DISCUSSION TITLE
Use as an API

### DISCUSSION THREAD

**Discussion by CesarVonc** [hiukim/mind-ar-js#259] [Q&A]
Hello,

Is it possible to use this wonderful library as an API, without Aframe or ThreeJS ? I don't see examples for this purpose.

For example, with the image tracker : give a webcam or video stream and get the 3D coordinates of the tracked image ?

Thanks

**Comment by hiukim** [comment]
It's a very good feature, but unfortunately not for now.

**Comment by kintovt** [comment]
https://ar-js-org.github.io/AR.js-Docs/ui-events/
At the bottom of the page you can find ar.js example of how to get camera/object coordinates.

We have targetFound and targetLost event in mindar too
https://hiukim.github.io/mind-ar-js-doc/examples/events-handling

so I tried to get coordinates in js but object position values are always set to 0,0,0
the same shows a-frame editor (ctrl+alt+i)

there are still some data like world matrix, camera matrix etc in a-entities js object. idk if it will be useful in getting target position, rotation and scale
